### PR TITLE
[Console] Improve OutputInterface PHPDoc

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Improve DX by mentioning constants. (Sometime we see only phpdoc in IDE but we don't open file, or scroll to top of file)

### CODE REVIEW

File: src/Symfony/Component/Console/Output/OutputInterface.php [hunk 1]
```diff
@@ -49,13 +49,15 @@ public function writeln($messages, int $options = 0);
 
     /**
      * Sets the verbosity of the output.
+     * 
+     * @param int $level One of self::VERBOSITY_* constants
```
Comment (GromNaN) [inline src/Symfony/Component/Console/Output/OutputInterface.php]: Psalm and Phpstan have a [notation for constant enumerations](https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants):

```suggestion
     * @param self::VERBOSITY_* $level
```

File: src/Symfony/Component/Console/Output/OutputInterface.php [hunk 2]
```diff
@@ -49,13 +49,15 @@ public function writeln($messages, int $options = 0);
...
      */
     public function setVerbosity(int $level);
 
     /**
      * Gets the current verbosity of the output.
      *
-     * @return int
+     * @return int One of self::VERBOSITY_* constants
```
Comment (GromNaN) [inline src/Symfony/Component/Console/Output/OutputInterface.php]:
```suggestion
     * @return self::VERBOSITY_*
```

**Comment by stof** [comment]
In the past, we did such kind of improvements in types in minor versions, not in patch versions when the existing type was not broken.

**Comment by fabpot** [comment]
That's for 6.4.

File: src/Symfony/Component/Console/Output/OutputInterface.php (hunk 1 above)
Comment (alamirault) [reply src/Symfony/Component/Console/Output/OutputInterface.php]: Great !

**Review by lyrixx — APPROVED** [review]

**Review by GromNaN — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alamirault.

**Pull request merged**